integration/services/ha_backend/init

```diff
@@ -33,22 +34,16 @@ ha_backend_setup() {
         return 1
     fi
 
-    hab pkg exec core/openssl openssl req -x509 -nodes -days 3650 -newkey rsa:2048 \
```

Comment:
Is it possible to generate the certificate and password and use that instead of relying on whatever the default is